src/zerocash/Coin.cpp

```diff
@@ -73,7 +47,8 @@ Coin::Coin(const std::string bucket, Address& addr): addr_pk(), cm(), rho(ZC_RHO
     this->rho = rho_v;
     this->addr_pk = addr.getPublicAddress();
 
-    std::vector<unsigned char> a_pk = addr.getPublicAddress().getPublicAddressSecret();
```

Comment:
(Expand the diff to see the lines I'm talking about). The order in the spec is (v, rho, r), but above it's (v, r, rho).